cmd/src/actions_exec.go

```diff
@@ -161,6 +164,9 @@ Format of the action JSON files:
 			return errors.New("cache is not a valid path")
 		}
 
+		if *clearCacheFlag {
```

Comment:
Hm?

Reply:
Compiler optimization: if I check for the flag, the compiler will more likely put the boolean into a register so the check will be faster down the line.

Reply:
(Just kidding of course, I forgot to remove this 😛 )

Reply:
The world of Go is endless 🙃